src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -85,9 +104,12 @@ public Task<int> Start()
 
                 if (!shouldSpawnNewProcess)
                 {
-                    return startTask.Task;
+                    return startTask.Task.IsCompleted
```

Comment:
If this method gets called 10 times before the server starts, all 10 threads will receive a response that this is a new process. The 11th thread will receive a response that this is not a process.

It will be up to the callers to handle locking the init for the first 10 calls.